### PR TITLE
Also send a mail to the object's owner when a MediaWallpost is created.

### PR DESCRIPTION
**Pull request by eodolphi**
Make sure we do not send the message when a donation is attached.

BB-14343 #resolve

### CODE REVIEW

File: bluebottle/wallposts/mails.py [hunk 1]
```diff
@@ -41,15 +41,16 @@
...
 from bluebottle.wallposts.notifiers import ObserversContainer
 
 logger = logging.getLogger(__name__)
 
 
 @receiver(post_save, weak=False, sender=TextWallpost)
+@receiver(post_save, weak=False, sender=MediaWallpost)
 def new_wallpost_notification(sender, instance, created, **kwargs):
-    if created:
+    if created and not instance.donation:
```
Comment (gannetson) [inline bluebottle/wallposts/mails.py]: A `MediaWallpost` won't ever have a donation, right?

Comment (eodolphi) [reply bluebottle/wallposts/mails.py]: Apparently it does. If I do a donation and then post a message, a mediawallpost is created.

Comment (eodolphi) [reply bluebottle/wallposts/mails.py]: I do not think we ever make textwallposts anymore.

Comment (gannetson) [reply bluebottle/wallposts/mails.py]: I guess you want `sender=Wallpost` here. Now you won't get notified when someone writes a `TextWallpost`.
I think a donation wallpost is a `SystemWallpost` unless someone adds a message, then it turns into a `TextWallpost`.

**Pull request merged**